week1_intro/primer/recap_ml.ipynb

```diff
@@ -709,9 +713,9 @@
    "source": [
     "### Part III: Numpy and vectorized computing\n",
     "\n",
-    "Almost any machine learning model requires some computational heavy lifting usually involving linear algebra problems. Unfortunately, raw Python is terrible at this because each operation is interpreted at runtime. \n",
+    "Almost any machine learning model requires some computational heavy lifting, usually involving linear algebra. Unfortunately, raw Python is terrible at this, because each operation is interpreted at runtime. \n",
     "\n",
-    "So instead, we'll use `numpy` - a library that lets you run blazing fast computation with vectors, matrices and other tensors. Again, the god object here is `numpy.ndarray`:"
+    "So instead, we'll use `numpy` - the library, that lets you run blazing fast computation with vectors, matrices and other tensors. Again, the god object here is `numpy.ndarray`:"
```

Comment:
I think both of these changes aren't OK: the library is being introduced here (hence "a"), and my English language model parses this comma as redundant, even though I can't point to a specific rule that governs this case.